src/components/About/AboutEnabledModules.js

```diff
@@ -33,7 +33,8 @@ class AboutEnabledModules extends React.Component {
 
       return (
         <li key={key} style={style}>
-          {this.props.availableModules[key]} (<tt>{key}</tt>)
+          {this.props.availableModules[key]}
+          {`(<tt>${key}</tt>)`}
```

Comment:
Because of the string interpolation, I think there might be issues with this possibly being escapes internally in React (`dangerouslySetInnerHTML={{ _html }}` exists for this reason).

Reply:
Fixed all of these.